llvm/test/tools/llvm-objcopy/tool-options.test

```diff
@@ -0,0 +1,4 @@
+## An error must be reported if a required argument value is missing.
+# RUN: not llvm-objcopy --only-section 2>&1 | FileCheck --check-prefix=CHECK-NO-VALUE %s
```

Comment:
nit: since at the moment there is only one check prefix in the entire test you can simply use the default one (CHECK)

Reply:
I agree, but I thought this test could be expanded in the future and in that case, it will be possible to add other patterns without rewriting existing ones.

Reply:
Sorry it was too strong. It is certainly possibly to add new check prefixes without rewriting the default one, but then the default one would stand out awkwardly, in my opinion.

Reply:
Unless you have explicit plans to extend the test in the near future, go with the simpler approach now (default prefix). If we need to change it in the future, we can.